api/v1/flow/flow.proto

```diff
@@ -464,6 +464,7 @@ message Policy {
 	string namespace = 2;
 	repeated string labels = 3;
 	uint64 revision = 4;
+    string kind = 5;
```

Comment:
This seems to be using spaces for indentation, where was the rest of the file uses tabs

Reply:
Ah, thanks. I'll push that change when I resolve the other comment.